doc/vstgui-dev.md

```diff
@@ -0,0 +1,66 @@
+# Developing with our VSTGUI Fork
+
+We made the hard choice [here](https://github.com/surge-synthesizer/surge/issues/515) to fork
+vstgui so we can maintain our own fixes while also submitting them to steinberg
+
+## How it is set up
+
+* [`surge-synthesizer/vstgui`](https://github.com/surge-synthesizer/vstgui) is a fork 
+of [`steinbergmedia/vstgui`](https://github.com/steinbergmedia/vstgui) with default branch set to master
+* [`surge-synthesizer/vstgui:surge`](https://github.com/surge-synthesizer/vstgui/tree/surge) is a branch 
+from [`surge-synthsizer/vstgui:master`](https://github.com/surge-synthesizer/vstgui/tree/master) with changes we have applied along the way
+* [`surge-synthesizer/surge`](https://github.com/surge-synthesizer/surge) has a submodule `vstgui.surge` which points to the `surge` branch of `vstgui`
```

Comment:
looks good, I would only change the wording near the end of this sentence. from:

> points to the `surge` branch of `vstgui`

to:

> points to the `surge` branch of our `vstgui` fork

Reply:
Great changed that and force pushed.